explore/src/clue/scala/queries/common/AsterismQueriesGQL.scala

```diff
@@ -20,4 +20,25 @@ object AsterismQueriesGQL {
       }
     """
   }
+
+  // Group these 2 mutations together so that the targetEdit and observationEdit subscriptions come
```

Comment:
Is this guaranteed to work?

Reply:
Well, we reduce the transit time involved, but they still aren't real close together. That timing depends on when the database updates happen for the 2 mutations, I guess. Still seemed better than doing them individually. 🤷‍♂️